### ISSUE TITLE
"TypeError: Object.values is not a function" on Node 6

### ISSUE THREAD

**Issue by amercier** [jest-community/jest-extended#150]
**Bug**

- `package` version: 0.8.0
- `node` version: 6.14.3 (LTS, [end-of-life April 2019](https://github.com/nodejs/Release))
- `npm` (or `yarn`) version: 0.33.11

Relevant code or config

```javascript
require('jest-extended');
```

What you did:

Require `jest-extended` on Node 6.

What happened (please provide anything you think will help):

```
 FAIL  src/index.spec.js
  ● Test suite failed to run
    TypeError: Object.values is not a function
      at Object.<anonymous> (node_modules/jest-extended/dist/matchers/index.js:269:26)
      at Object.<anonymous> (node_modules/jest-extended/dist/index.js:3:17)
```

**Comment by benjaminkay93** [comment]
Ahh, Object.values only got introduced in node 7, Looks like it got brought in with https://github.com/jest-community/jest-extended/pull/142

**Comment by benjaminkay93** [comment]
We could swap this out for 
``` js
export default Object.keys(imports).map(key => imports[key]).reduce...
```

**Comment by mattphillips** [comment]
Thanks @amercier for raising this, I've just published [v0.8.1](https://www.npmjs.com/package/jest-extended/v/0.8.1) with @benjaminkay93's fix :smile:
